library/hawkular_alerts_group_trigger.py

```diff
@@ -122,37 +122,85 @@ def delete_group_trigger(self, group_id):
         except Exception as e:
             self.module.fail_json(msg="Failed to delete group trigger. Error: {error}".format(error=e))
 
-    def create_group_trigger(self, name, group_id, severity, enabled):
-        """ Creates a group trigger in Hawkular Alerting component
+    def required_updates(self, trigger, name, severity, enabled):
+        """ Checks whether an update is required for the group trigger
+
+            Returns:
+                True if the passed trigger name, severity or enabled status
+                are different from the ones set in the trigger, False otherwise
+        """
+        updates = {}
+        if (name and trigger.name != name):
```

Comment:
the left half is to allow omitted module params to not modify the trigger, right?

can any of those be 0, "", or False? (especially `enabled: false` in yaml?)
I think in that case this would erronously decide they don't need updating.